typegen/strats/__init__.py

```diff
@@ -1,7 +1,11 @@
 from .gen import TypeHintGenerator
-from .inline import InlineGenerator
+from .inline import EvaluationInlineGenerator, InlineGenerator, RemoveAllTypeHintsTransformer
```

Comment:
Why export the `RemoveAllTypeHintsTransformer`?

Reply:
Removed